### PR TITLE
Added Font Awesome

### PR DESCRIPTION
**Pull request by RichardLitt**
I believe this closes https://github.com/opensourcedesign/opensourcedesign.github.io/issues/5.

### CODE REVIEW

**Comment by Xaviju** [comment]
Thanks for the PR
I think https://github.com/opensourcedesign/opensourcedesign.github.io/issues/5 was supposed to add the icon to the website dependencies, not to the resources. 
Anyway, regarding resource, is font-awesome totally Open Source?

**Comment by RichardLitt** [comment]
Will comment on the linked issue independently. 

It is _mostly_ OS. The company that runs it has some close sourced material, but all of the FA stuff you see on the Website is MIT license, all of the documentation CC BY 3.0.

**Comment by simonv3** [comment]
Huh! Out of curiosity - what is the closed source material?

**Comment by RichardLitt** [comment]
Private repo access, as well as their internal SVG tooling, isn't. You can get that by look at their kickstarter. https://www.kickstarter.com/projects/232193852/font-awesome-5. That having been said, I think this is about as closed source as my analog notebooks are - everything they produce basically becomes OS, unless it is for a client. One of my favorite companies, in that regard.

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Commons.
 source icon sets to use as Inkscape symbols.
 * [IconMonstr](http://iconmonstr.com/) A large collection of glyph icons from a 
 German artist. Free to use without attribution and free to modify.
-
+* [FontAwesome](http://fontawesome.io/icons/) The iconic font and CSS toolkit. Free to use.
```
Comment (jdorfman) [inline README.md]: Maybe mention https://www.bootstrapcdn.com/fontawesome/ ?

**Comment by Xaviju** [comment]
Thanks @RichardLitt I'd approve this PR as is.

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Added a link to the CDN and GitHub.

**Comment by RichardLitt** [comment]
@Xaviju Feel free to merge. :)

**Pull request merged**